.github/actions/update-check/action.yml

```diff
@@ -83,21 +83,31 @@ runs:
             exclude_pull_requests: true
           });
 
-          const message = "The CI workflow run with tests that require additional secrets finished as " + process.env.conclusion + ": " + run.html_url
+          const started = "The CI workflow run with tests that require additional secrets has been started: " + run.html_url
+          const finished = "The CI workflow run with tests that require additional secrets finished as " + process.env.conclusion + ": " + run.html_url
           const comments = await github.paginate(github.rest.issues.listComments.endpoint.merge({
             ...context.repo,
             issue_number: process.env.number
           }))
```

Comment:
unrelated but can we only list for "recent" comments instead of all comments on every invocation

Reply:
I'm not sure how that'd help and if it's possible at all. I don't see a method for that in https://octokit.github.io/rest.js/v19#issues-list-comments

Reply:
I was thinking about rate-limits but it appears we never hit those. There is a `since` parameter.